picard/log.py

```diff
@@ -165,11 +104,9 @@ def clear(self):
 
 def name_filter(record):
     # provide a significant name, because module sucks
-    prefix = os.path.dirname(os.path.normcase(_srcfile))
-    name = record.pathname
-    if name.startswith(prefix):
-        name = name[len(prefix) + 1:].replace(os.path.sep, '.').replace('.__init__', '')
-    record.name, _ = os.path.splitext(name)
+    name = pathlib.Path(record.pathname)
```

Comment:
It took me a while to figure out what this function does - I see that it strips `picard` from the beginning of a path, removes the file extension and any occurences of `.__init__` and for plugins adds the whole path to the file at the beginning of the line, right? In that case, sure, looks reasonable.

Note that before this change, plugin logs were like

> /home/wieland/.config/MusicBrainz/Picard/plugins/keep.transltag:19:

now they're like

>  home.wieland..config.MusicBrainz.Picard.plugins.keep.transltag:19:

Reply:
> Note that before this change, plugin logs were like
> 
>     /home/wieland/.config/MusicBrainz/Picard/plugins/keep.transltag:19:
> 
> now they're like
> 
>     home.wieland..config.MusicBrainz.Picard.plugins.keep.transltag:19:

Not very fancy, that's only for plugins right ?
Can you post a bigger part of your log ?